services/common/src/interfaces/reports/mineReportPageData.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { IMineReport } from "@mds/common/interfaces";
+
+export interface IMineReportPageData {
+  records: IMineReport[];
```

Comment:
oh! We do have a generic IPageData<T> interface that can be used with IPageData<IMineReport> instead of creating a new interface if you like.

Reply:
Oooh, nice!